README.md

```diff
@@ -4,6 +4,8 @@ Logo design by [Roy Quilor](https://www.quilor.com/) is licensed under [CC BY-NC
 
 # Archivy
 
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```

Comment:
I don't think we need this to be put so prominently on the README, could you remove it?